fixtures/foreign-executor/tests/test_generated_bindings.rs

```diff
@@ -1,5 +1,8 @@
 uniffi::build_foreign_language_testcases!(
     "tests/bindings/test_foreign_executor.py",
-    "tests/bindings/test_foreign_executor.kts",
-    "tests/bindings/test_foreign_executor.swift",
+    // Disable due to the Docker image being outdated for now.
+    // Please see https://github.com/mozilla/uniffi-rs/pull/1409#issuecomment-1437170423
+    //
+    // "tests/bindings/test_foreign_executor.kts",
+    // "tests/bindings/test_foreign_executor.swift",
```

Comment:
This is really annoying :-(

Reply:
If I don't get the mozilla/uniffi-ci image enabled by the end of the week, we can move over to another user-owned container for the time being.

Reply:
:+1:

Reply:
\o/